From d2ddf9a441fb42ad6025d11cdbb117a879b9b12f Mon Sep 17 00:00:00 2001
From: Richard Danter <radanter@gmail.com>
Date: Fri, 13 Oct 2023 12:04:56 +0100
Subject: [PATCH] Fix drm panel flag name

Previously a patch was included which added a flag to indicate that
the "upstream" bridge should be enabled before the panel, etc.

In RCPL 2 the patch was already included, but it used a different
name, "prev" instead of "upstream".

This patch fixes the name where it was added by other patches.

Signed-off-by: Richard Danter <radanter@gmail.com>
---
 drivers/gpu/drm/bridge/panel.c                | 4 ++--
 drivers/gpu/drm/bridge/tc358762.c             | 2 +-
 drivers/gpu/drm/panel/panel-ilitek-ili9881c.c | 2 +-
 include/drm/drm_panel.h                       | 6 +++---
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/drivers/gpu/drm/bridge/panel.c b/drivers/gpu/drm/bridge/panel.c
index 0c1dcf85ed13..d69986a74e7f 100644
--- a/drivers/gpu/drm/bridge/panel.c
+++ b/drivers/gpu/drm/bridge/panel.c
@@ -258,8 +258,8 @@ struct drm_bridge *drm_panel_bridge_add_typed(struct drm_panel *panel,
 	panel_bridge->bridge.ops = DRM_BRIDGE_OP_MODES;
 	panel_bridge->bridge.type = connector_type;
 
-	panel_bridge->bridge.pre_enable_upstream_first =
-						panel->prepare_upstream_first;
+	panel_bridge->bridge.pre_enable_prev_first =
+						panel->prepare_prev_first;
 
 	drm_bridge_add(&panel_bridge->bridge);
 
diff --git a/drivers/gpu/drm/bridge/tc358762.c b/drivers/gpu/drm/bridge/tc358762.c
index 9914cfb05b11..35c03e1939b0 100644
--- a/drivers/gpu/drm/bridge/tc358762.c
+++ b/drivers/gpu/drm/bridge/tc358762.c
@@ -229,7 +229,7 @@ static int tc358762_probe(struct mipi_dsi_device *dsi)
 	ctx->bridge.funcs = &tc358762_bridge_funcs;
 	ctx->bridge.type = DRM_MODE_CONNECTOR_DPI;
 	ctx->bridge.of_node = dev->of_node;
-	ctx->bridge.pre_enable_upstream_first = true;
+	ctx->bridge.pre_enable_prev_first = true;
 
 	drm_bridge_add(&ctx->bridge);
 
diff --git a/drivers/gpu/drm/panel/panel-ilitek-ili9881c.c b/drivers/gpu/drm/panel/panel-ilitek-ili9881c.c
index dfb90334a77c..474743832b6c 100644
--- a/drivers/gpu/drm/panel/panel-ilitek-ili9881c.c
+++ b/drivers/gpu/drm/panel/panel-ilitek-ili9881c.c
@@ -1352,7 +1352,7 @@ static int ili9881c_dsi_probe(struct mipi_dsi_device *dsi)
 	ctx->dsi = dsi;
 	ctx->desc = of_device_get_match_data(&dsi->dev);
 
-	ctx->panel.prepare_upstream_first = true;
+	ctx->panel.prepare_prev_first = true;
 	drm_panel_init(&ctx->panel, &dsi->dev, &ili9881c_funcs,
 		       DRM_MODE_CONNECTOR_DSI);
 
diff --git a/include/drm/drm_panel.h b/include/drm/drm_panel.h
index 451f88efcd1c..bffb1efa5daf 100644
--- a/include/drm/drm_panel.h
+++ b/include/drm/drm_panel.h
@@ -197,14 +197,14 @@ struct drm_panel {
 	struct list_head list;
 
 	/**
-	 * @prepare_upstream_first:
+	 * @prepare_prev_first:
 	 *
-	 * The upstream controller should be prepared first, before the prepare
+	 * The previous controller should be prepared first, before the prepare
 	 * for the panel is called. This is largely required for DSI panels
 	 * where the DSI host controller should be initialised to LP-11 before
 	 * the panel is powered up.
 	 */
-	bool prepare_upstream_first;
+	bool prepare_prev_first;
 };
 
 void drm_panel_init(struct drm_panel *panel, struct device *dev,
-- 
2.34.1

